collections/JAXA_wq_chla.yaml

```diff
@@ -0,0 +1,28 @@
+Name: JAXA_wq_chla
+Description: "water quality Chlorophil a anomalies"
```

Comment:
Suggestion: change *Chlorophil* to Chlorophyll for consistency

Reply:
done